Pod/Classes/WPImageExporter.h

```diff
@@ -0,0 +1,9 @@
+#import <Foundation/Foundation.h>
+
+@interface WPImageExporter : NSObject
```

Comment:
Could you write a quick doc comment explaining what this class is for / why we need it?